### PR TITLE
add datasets

### PR DESCRIPTION
**Pull request by mmahbub**

### CODE REVIEW

**Comment by sudarshan85** [comment]
Could you please provide a description? Which document did you add in the list? Did you  run the data setup code and verify that the length of the new datasets created in the `huggingface_hub` has increased by the exact amount of the sentences you added?

**Comment by mmahbub** [comment]
Yes, I checked all those before creating the pull request.

I added these 3 documents in total: "the_former_yugoslav_republic_of_macedonia_early_parliamentary_elections_5_june_2016:_interim_report_1", 
"former_yugoslav_republic_of_macedonia_early_parliamentary_elections_5_june_2011:_interim_report_no._1", 
"guidelines_for_applicants". 

For these documents, I added a total of 136+155+172=463 lines per language. 
So the test files in the `huggingface_hub` had 605 lines before, but now it has 605+463=1068 lines.

One thing to note here. You may have done this on purpose, but I just wanted to bring your attention to it if you didn't. Because of the seed that you have set in the ` data_setup.py`, every time the code creates a split of train-test at index 5238. So now when I add more data, they get appended to the test files.

**Comment by sudarshan85** [comment]
> One thing to note here. You may have done this on purpose, but I just wanted to bring your attention to it if you didn't. Because of the seed that you have set in the data_setup.py, every time the code creates a split of train-test at index 5238. So now when I add more data, they get appended to the test files. I didn't know if you wanted me to use a different seed, so I didn't change it at runtime.

Nice catch! Thank you for spotting that bug. I've issued a fix. Now the first 15% (or p% specified by the user) is always the test set and the rest is the training set. Could you please sync with remote and run the dataset up again and update the pull request.

**Comment by mmahbub** [comment]
After fixing there are 946 test samples (previously there were 877) and 5360 train samples (previously there were 4966). In total, we have 69+394=463 new samples per language.

**Comment by mmahbub** [comment]
Should I provide the stat each time I create a pull request? Or was it just a one-time-only thing?

**Comment by sudarshan85** [comment]
As long as the numbers make match up and make sense to you, you don't need to specify it each time.
